learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -87,55 +108,66 @@ $ bal grpc --mode service --input helloworld.proto --output greeter_service/
 Once successfully executed, you will see the output below.
 
 ```bash
+Continuing with the existing protoc executor file at /var/folders/6p/8h4k83hj0r98n1s6mc3jy0480000gn/T/protoc-3.9.1-osx-x86_64.exe
 Successfully extracted the library files.
 Successfully generated the Ballerina file.
 ```
 
-Once you move into the directory, you will see the following two files,
+This creates two files below inside the `greeter_service` directory.
 
 ```bash
-greeter_service
-├── - - -
+.
+├── Ballerina.toml
```

Comment:
Avoid listign `Ballerina.toml` here as it was created in previous step and  not by this command. And the text  above is also saying `This creates two files below` but we are showing 3 files here.

Reply:
Done.